src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -186,17 +92,16 @@ public boolean doMonitor(ServiceEmitter emitter)
           .setDimension("poolName", pool.getName());
       MonitorUtils.addDimensionsToBuilder(builder, dimensions);
 
-      emitter.emit(builder.build("jvm/pool/max",       usage.getMax()));
+      emitter.emit(builder.build("jvm/pool/max", usage.getMax()));
       emitter.emit(builder.build("jvm/pool/committed", usage.getCommitted()));
-      emitter.emit(builder.build("jvm/pool/used",      usage.getUsed()));
-      emitter.emit(builder.build("jvm/pool/init",      usage.getInit()));
+      emitter.emit(builder.build("jvm/pool/used", usage.getUsed()));
+      emitter.emit(builder.build("jvm/pool/init", usage.getInit()));
     }
+  }
 
-    // jvm/gc
-    emitGcMetrics(youngGcCounters, emitter);
-    emitGcMetrics(oldGcCounters, emitter);
+  private void emitDirectMemMetrics(ServiceEmitter emitter)
+  {
 
-    // direct memory usage
     for (BufferPoolMXBean pool : ManagementFactory.getPlatformMXBeans(BufferPoolMXBean.class)) {
```

Comment:
Don't think we need it because `BufferPoolMXBean`s are cached and only `unmodifiableList` of these objects  is created on `getPlatformMXBeans` method call, so I don't think it's a big problem for us.